com.ibm.wala.cast.java.test/src/com/ibm/wala/cast/java/test/JavaIRTests.java

```diff
@@ -82,103 +82,78 @@ public JavaIRTests() {
   @Test public void testTwoClasses() throws IllegalArgumentException, CancelException, IOException {
     runTest(singleTestSrc(), rtJar, simpleTestEntryPoint(), Arrays.asList(
 
-    new IRAssertion() {
-
-      @Override
-      public void check(CallGraph cg) {
-        final String typeStr = singleInputForTest();
+    (IRAssertion) cg -> {
```

Comment:
@liblit Why do we need the `(IRAssertion)` cast here?

Reply:
We don’t. I will remove the unnecessary cast here and at several other places in this same file. Thank you!